schema.graphql

```diff
@@ -154,10 +154,10 @@ type BondDiscount @entity {
   id: ID!
   timestamp: BigInt!
   dai_discount: BigDecimal!
-  ohmdai_discount: BigDecimal!
+  brickdai_discount: BigDecimal!
```

Comment:
there is no brickdai and bricklusd right?

Reply:
yes, can be removed instead